app/authenticated/projects/edit/route.js

```diff
@@ -0,0 +1,16 @@
+import { hash } from 'rsvp';
+import { inject as service } from '@ember/service';
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+    authzStore: service('authz-store'),
```

Comment:
Not sure what editor you're using but in this project indenting is 2 spaces (not 4, or a tab)..